compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -295,12 +305,15 @@ static LogicalResult setRootConfigForPackPeelPipeline(
   // ------------------------------------------------------
   MLIRContext *context = entryPointFn.getContext();
 
-  // Transpose B matrix from [K N n k] to [K N k n]
+  // For matmul, transpose B matrix from [K N n k] to [K N k n]
+  // For matmul_transpose_b, we don't have to transpose the B matrix,
+  // since it is already [N K n k]
   SmallVector<int64_t> transposePackIndices = {1};
   // There is no corresponding unpack for the specified pack operation
   // 0 is used when unpack is empty
   SmallVector<bool> unpackEmpty = {false};
-  SmallVector<SmallVector<int64_t>> innerPerm = {{1, 0}};
+  SmallVector<int64_t> innerPermB = setInnerPermB(isMatmulTranspose);
```

Comment:
nit: could do without function call:

SmallVector<int64_t> innerPermB  =  isMatmulTranspose ? {0,1} : {1,0};

Reply:
It doesn't work to initialize the vector with constructors in this way. It has to be static during the compile time.

Reply:
Ok. I think the function can be simplified, and maybe should be called 'get' rather than 'set'.